wordpress-mobile/WordPress-iOS
# Themes: Fix theme url to include the correct image resized url

This PR fixes the URL that we genetate to display the theme screenshots. 

Before: https://i2.wp.com/s2.wp.com/wp-content/themes/pub/blank-canvas/screenshot.png?ssl=1?w=409
After: https://i2.wp.com/s2.wp.com/wp-content/themes/pub/blank-canvas/screenshot.png?ssl=1&w=409

The fix should download smaller images for each of the themes saving bandwidth and mobile data.
 
This issue was discoverd while trying to clear the screenshot image cache for the blank-canvas theme. 

To test:
Navigate to the theme selection screen for a .com site. Notice that the screenshots show up as expected.
.org sites do not have the theme selection as an option.
Navigate to an atomic and jetpack site notice that the themes show up as expected. 

## Regression Notes
1. Potential unintended areas of impact
None that I can think of

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
